src/Native/System.Net.Http.Native/pal_easy.h

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "pal_types.h"
+
+#include <curl/curl.h>
+
+enum
+{
+    CurlOptionLongBase = 0,
+    CurlOptionObjectPointBase = 10000,
+    CurlOptionFunctionPointBase = 20000,
+};
+
+enum PAL_CURLoption : int32_t
+{
+    PAL_CURLOPT_INFILESIZE = CurlOptionLongBase + 14,
```

Comment:
@vijaykota The C# code which calls into here needs to specify the value explicitly.  The PAL enums therefore specify their types explicitly to match.  Then the static_assert verifies that our belief that curl hasn't changed their underlying values holds true (therefore asserting that the C# version and the curl version are the same, so we don't need a mapping function).

If we used the curl.h values directly we wouldn't know if the numbers changed out from under us, except that tests would start failing in weird ways.

Not that curl is known to have done that, but we're being defensive in all our Unix interop.